src/grid/BindingColumn.php

```
@@ -19,72 +14,163 @@
 
-    public function init()
+    public function init(): void
     {
         parent::init();
         $this->visible = Yii::$app->user->can('hub.read');
-        $this->label = $this->getLabel();
+        $this->label = $this->buildLabel();
     }
 
-    public function getDataCellValue($model, $key, $index)
+    public function getDataCellValue($model, $key, $index): string
```

PHPMD (2.15.0)

Avoid unused parameters such as '$key'. (Unused Code Rules)

(UnusedFormalParameter)